optaweb-vehicle-routing-backend/pom.xml

```diff
@@ -55,6 +55,11 @@
       <artifactId>h2</artifactId>
       <scope>runtime</scope>
     </dependency>
+    <dependency>
```

Comment:
Is it possible to connect the backend container to another container (which is the postgres database) provisioned by the ./provision script?

Reply:
Yes of course. On OpenShift, the backend and postgresql containers run in different pods. Backend accesses the database pod through a Service called `postgresql`. The service is a result of `oc new-app postgresql-persistent`. I don't specify a name for the app so the Service will be named `postgresql`.

The service name is then hard-coded and consumed in backend's `application.properties`.

Reply:
https://dzone.com/articles/configuring-spring-boot-for-postgresql

Reply:
Odd, I cannot mark this comment was resolved; so this comment has to do:

MARK AS RESOLVED BY @Christopher-Chianelli